images/tang/Containerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/centos/centos:stream8
+FROM quay.io/centos/centos:stream9
```

Comment:
you may need to change this in the prow config first

Reply:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master.yaml#L45